scripts/global.js

```diff
@@ -921,52 +923,81 @@ export async function restoreWallet() {
     }
 }
 
+/**
+ * Fetch Governance data and re-render the Governance UI
+ */
 async function updateGovernanceTab() {
-    const proposals = await Masternode.getProposals();
-    doms.domGovProposalsTableBody.innerHTML = '';
-    for (const proposal of proposals) {
-        if (proposal.RemainingPaymentCount === 0) {
-            continue;
-        }
-        const tr = doms.domGovProposalsTableBody.insertRow();
-        const td1 = tr.insertCell();
-        // IMPORTANT: We must sanite all of our HTML or a rogue server or malicious proposal could perform a cross side scripting attack
-        td1.innerHTML = `<a class="active" href="${sanitizeHTML(
-            proposal.URL
-        )}"><b>${sanitizeHTML(proposal.Name)}</b></a>`;
-        const td2 = tr.insertCell();
-        td2.innerHTML = `<b>${sanitizeHTML(proposal.MonthlyPayment)}</b> ${
+    // Fetch all proposals from the network
+    const arrProposals = await Masternode.getProposals();
+
+    /* Sort proposals into two categories
+        - Standard
+        - Contested (When a proposal may be considered spam, malicious, or simply highly contestable)
+    */
+    const arrStandard = arrProposals.filter(a => a.RemainingPaymentCount > 0 && a.Ratio > 0.25);
+    const arrContested = arrProposals.filter(a => a.RemainingPaymentCount > 0 && a.Ratio <= 0.25);
```

Comment:
Without enough statistics it does not make any sense to call a proposal contested.  I'd add the condition that the total number of votes is at least 100 (or any big enough number)

Reply:
I'd also suggest renaming the section to "Other Proposals" as _contested_ is subjective and will possibly not be true for all conditions.

Reply:
Good catch on a 'minimum' of votes, panleone - adding this! :+1:
I think 100 votes minimum is fine.

@shrnkld The "subtext" for the category explains that it's not a "deterministic" decision, just a kind of catch-all net for anything that may be **likely** spam or 'contestable'.

I also would rather not go against what the community voted for; as the choice was given to them.